spmd/tensor/dispatch.py

```diff
@@ -111,20 +111,12 @@ def _reshape_alias(
 ] = {torch.ops.aten._reshape_alias.default: _reshape_alias}
 
 
-def operator_dispatch(
+def prepare_inputs(
     op_call: torch._ops.OpOverload,
     args: Tuple[object, ...],
     kwargs: Dict[str, object],
     op_to_rules: Dict[str, Callable[[OpSchema], OutputSharding]],
-    custom_dispatch_ops: Dict[str, Callable[..., object]],
-) -> object:
-    # first we need to lift some private aten aliases to public calls
-    if op_call in _CURRENT_DECOMPOSITION_TABLE:
-        return _CURRENT_DECOMPOSITION_TABLE[op_call](*args, **kwargs)
-
-    func_schema = FunctionSchema.parse(str(op_call._schema))
-    schema_kind = func_schema.kind()
-
+) -> Tuple[OpSchema, bool, Optional[OutputSharding]]:
```

Comment:
what is the second bool return argument mean?

Reply:
It should be the `redistributed` bool in the original implementation. IIUC, that dictates whether we distributed input tensors?